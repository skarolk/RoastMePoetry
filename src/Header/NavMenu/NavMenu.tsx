import React from "react";
import Navbar from "react-bootstrap/Navbar";
import Button from "react-bootstrap/Button";
import Nav from "react-bootstrap/Nav";

const NavMenu = (props: any) => {
  return (
    <Navbar bg="dark" variant="dark" sticky="top">
      <Navbar.Brand href="#home">
        <img
          alt=""
          src="https://react-bootstrap.github.io/logo.svg"
          width="30"
          height="30"
          className="d-inline-block align-top"
        />{" "}
        RoastMePoetry
      </Navbar.Brand>
      <Nav className="justify-content-end" style={{ width: "100%" }}>
        <Button variant="outline-info">Buy Now</Button>
      </Nav>
    </Navbar>
  );
};

export default NavMenu;
