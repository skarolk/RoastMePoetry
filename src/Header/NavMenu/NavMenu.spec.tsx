import React from "react";
import renderer from "react-test-renderer";
import NavMenu from "./NavMenu";

it("renders correctly", () => {
  const RenderedNavMenu = renderer.create(<NavMenu />).toJSON();
  expect(RenderedNavMenu).toMatchSnapshot();
});
