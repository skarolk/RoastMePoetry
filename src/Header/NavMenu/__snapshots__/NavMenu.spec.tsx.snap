// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="navbar navbar-expand navbar-dark bg-dark sticky-top"
>
  <a
    className="navbar-brand"
    href="#home"
  >
    <img
      alt=""
      className="d-inline-block align-top"
      height="30"
      src="https://react-bootstrap.github.io/logo.svg"
      width="30"
    />
     
    RoastMePoetry
  </a>
  <div
    className="justify-content-end navbar-nav"
    onKeyDown={[Function]}
    style={
      Object {
        "width": "100%",
      }
    }
  >
    <button
      className="btn btn-outline-info"
      disabled={false}
      type="button"
    >
      Buy Now
    </button>
  </div>
</nav>
`;
